apps/web/src/App.css

```
@@ -0,0 +1,261 @@
+.drawer-body {
+  padding: 1rem;
+  overflow-y: auto;
+}
+
+.muted {
+  color: var(--muted);
+}
+
+.markdown {
+  line-height: 1.6;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)